apps/frontend/src/components/cards/controltable/ControlRowCol.vue

```diff
@@ -21,41 +21,53 @@
     >
       <h3 class="pa-2">Test</h3>
       <v-divider />
-      <div class="pa-2 mono text-justify">{{ result.code_desc.trim() }}</div>
+      <!-- HTML is sanitized with sanitize-html -->
+      <!-- eslint-disable -->
```

Comment:
Can this be updated to disable a specific rule instead of globally disabling eslint for this line?